front/app/containers/InitiativesShow/ReactionControl/Status/index2.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box, Icon, IconNames } from '@citizenlab/cl2-component-library';
+import { StatusWrapper, StatusExplanation } from '../SharedStyles';
+import T from 'components/T';
+import { StatusComponentProps } from '.';
+import ReadAnswerButton from './components/ReadAnswerButton';
+import VoteButtons from './components/VoteButtons';
+import ReactionCounter from './components/ReactionCounter';
+import CountDown from '../CountDown';
+
+const StatusIcon = styled(Icon)`
+  path {
+    fill: ${(props) => props.theme.colors.tenantText};
+  }
+  width: 30px;
+  height: 30px;
+`;
+
+interface Props extends StatusComponentProps {
+  iconName: IconNames;
+  statusExplanation: React.ReactNode;
+  barColor?: string;
+  showCountDown: boolean;
+  showVoteButtons?: boolean;
+  cancelReactionDisabled?: boolean;
+  showReadAnswerButton?: boolean;
```

Comment:
Made all "show" props required in a later PR to make your choices very explicit, to not forget UI parts (+ you're able to understand what a component will do from seeing its passed props).